doc/guides/contributing/pull-requests.md

```diff
@@ -176,14 +176,10 @@ Sample complete commit message:
 ```txt
 subsystem: explain the commit in one line
 
-Body of commit message is a few lines of text, explaining things
-in more detail, possibly giving some background about the issue
-being fixed, etc.
-
-The body of the commit message can be several paragraphs, and
-please do proper word-wrap and keep columns shorter than about
-72 characters or so. That way, `git log` will show things
-nicely even when it is indented.
+The body of the commit message can be several paragraphs, explaining things in
+more detail, possibly giving some background about the issue being fixed, etc.
+Please do proper word-wrap and keep columns shorter than about 72 characters
+or so. That way, `git log` will show things nicely even when it is indented.
```

Comment:
Remove?

Reply:
@Trott Sorry, but what exactly should I remove here?

Reply:
The whole line? The previous line can end with a period and this line be removed. Just a suggestion. Feel free to ignore if you disagree with it.

Reply:
@Trott Done!